### PR TITLE
Added Web App workflow

### PR DESCRIPTION
**Pull request by Chukwuemeka-James**

### CODE REVIEW

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Add `.env.test` to populate telegram creds to pretend error there

**Review by djeck1432 — CHANGES_REQUESTED** [review]
You still have the same error, please fix that

https://github.com/Chukwuemeka-James/derisk-research/actions/runs/11962413333/job/33350769876

**Comment by djeck1432** [comment]
@Chukwuemeka-James Any progres?

File: .github/workflows/wep_app_ci.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+name: Web App CI Workflow
+
+on: [push, pull_request]
```
Comment (djeck1432) [inline .github/workflows/wep_app_ci.yml:3]:
```suggestion
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```

**Review by djeck1432 — CHANGES_REQUESTED** [review]

**Comment by djeck1432** [comment]
@Chukwuemeka-James Any progress?

**Comment by Chukwuemeka-James** [comment]
I'm just seeing this right now. I'll implement the suggested changes and
create a PR. Thank you for your patience 🙏🏿.

File: .github/workflows/wep_app_ci.yml (hunk 1 above)
Comment (djeck1432) [reply .github/workflows/wep_app_ci.yml:3]: why you didn't fix that?

File: .github/workflows/wep_app_ci.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          poetry install
+    
+      - name: Run Tests for Web App
+        working-directory: ./apps/web_app
+        run: |
+          echo "PATH=$HOME/.local/bin:$PATH" >> $GITHUB_ENV
+          cp .env.example .env
+          poetry run pytest
+     
+      - name: Install dependencies for Shared
```
Comment (djeck1432) [inline .github/workflows/wep_app_ci.yml:49]: why do you need here shared?

File: .github/workflows/wep_app_ci.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        working-directory: ./apps/shared
+        run: |
+          echo "PATH=$HOME/.local/bin:$PATH" >> $GITHUB_ENV
+          poetry lock --no-update
+          poetry install
+    
+      - name: Run Tests for Shared
```
Comment (djeck1432) [inline .github/workflows/wep_app_ci.yml:56]: Why do you need here run for shared?

**Review by djeck1432 — CHANGES_REQUESTED** [review]

**Comment by djeck1432** [comment]
@Chukwuemeka-James Any updates?

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**